### PR TITLE
Build using AdoptOpenJDK 8 & 11

### PR DESCRIPTION
**Pull request by marcospereira**
## Purpose

Build Play using AdoptOpenJDK 8 & 11 (11 still allowed to fail).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -55,22 +67,21 @@ matrix:
...
-  global:
-    secure: "NS2hMbBcmi6EF4QxtcNs4A2ZuNmIdLYQRJUWWejgnD4YtcsmoVjxrHRedqrnDdui4DyvaxWhg/3Uds23jEKTSbbh3ZphLO77BVgM2nUGUvVoa4i6qGF2eZFlIhq2G1gM700GPV7X4KmyjYi2HtH8CWBTkqP3g0An63mCZw/Gnlk="
+    - env: TRAVIS_JDK=adopt@1.11.0-1
 
 cache:
   directories:
-  - $HOME/.ivy2/cache
-  - $HOME/.sbt
+    - "$HOME/.ivy2/cache"
+    - "$HOME/.sbt/boot/"
```
Comment (renatocaval) [inline .travis.yml]: On template control, we are using this instead.
 
```
- "$HOME/.sbt"
```

This was a change suggested by @dwijnand and is now being used in all samples

**Comment by mkurz** [comment]
Should this be backported to `2.7.x`?

**Comment by dwijnand** [comment]
No it's not necessary.

**Comment by TimMoore** [comment]
If we don't backport the changes for Java 8, then it will mean 2.7.x will be tested with an outdated version of Java once the public updates of Oracle Java 8 stop.

**Comment by dwijnand** [comment]
Cherry-picked in 9ab496d8ad for 2.7.x and in #8942 for 2.6.x.

**Comment by dwijnand** [comment]
The cherry-pick to 2.7.x broke some documentation test in CI: https://travis-ci.org/playframework/playframework/jobs/479291791

```
[error] Test javaguide.ehcache.JavaEhCache.http failed: 
[error] Expected: "something else"
[error]      but: was "Hello world", took 0.119 sec
```

Is this a known issue?  Should we open a ticket for this or what's the regular procedure when a backport breaks CI?

**Comment by mkurz** [comment]
I think I saw that error before already... Maybe this just randomly fails because ehcache takes to long to cache "something else"? (Or similiar) Maybe we have to add a timeout before accessing ehcache to get that value?

**Pull request merged**